ui/server/src/test/scala/pl/touk/nussknacker/ui/definition/DefinitionPreparerSpec.scala

```diff
@@ -59,7 +59,7 @@ class DefinitionPreparerSpec extends FunSuite with Matchers with TestPermissions
 
     val baseNodes = baseNodeGroups.flatMap(_.possibleNodes)
     // 5 nodes from base + 3 custom nodes
-    baseNodes should have size (5 + 3)
+    baseNodes should have size (6 + 3)
```

Comment:
Please fix the comment above or maybe introduce  constants `baseNodesCount`, `customNodesCount`?